schematic/store/synapse.py

```diff
@@ -680,105 +704,44 @@ def getFilesInStorageDataset(
         Raises:
             ValueError: Dataset ID not found.
         """
-        # select all files within a given storage dataset folder (top level folder in
-        # a Synapse storage project or folder marked with contentType = 'dataset')
-        walked_path = synapseutils.walk(
-            self.syn, datasetId, includeTypes=["folder", "file"]
-        )
+        file_list = []
 
-        current_entity_location = self.synapse_entity_tracker.get(
-            synapse_id=datasetId, syn=self.syn, download_file=False
-        )
+        # Get path to dataset folder from fileview to avoid building a new fileview and walking to determine folders and files within
+        child_path = self.storageFileviewTable.loc[
+            self.storageFileviewTable["parentId"] == datasetId, "path"
+        ][0]
+        parent = child_path.split("/")[0]
+        dataset_path = f"{parent}/%"
+        dataset_path = "'" + dataset_path + "'"
```

Comment:
```suggestion
        dataset_path = f"'{parent}/%'"
```

Sorry I meant things like this is a bit awkward, but I think you can have single quotes like the above
```
f"{json['foo']}/mytext"
```

Reply:
Oh okay, I thought you meant to specifically avoid doing that. I understand now thanks